src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,151 @@
+package duke.util;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * duke.util.Ui class that deals with interactions with the user.
+ *
+ * @author Pearlynn
+ */
+
+public class Ui {
+    private final String line = "\t____________________________________________________________";
```

Comment:
Isn't this a constant? If it is, ~~line~~ LINE should be in uppercase.

Constants should also be static.